FileWriteFlags.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+namespace FileWriteFlag
```

Comment:
Do these flags apply to all file streams, or only write streams?

Reply:
I thought the name FileWriteFlag indicated fairly clearly indicate the flags only apply to FileWriteStreams. Although I'm guessing this was not intuitive to you since you asked the question. I'm not sure how to change the name to more clearly indicate this.

Reply:
I suppose I made a bad assumption there based on a lot of other systems that have a single set of file flags for both read and write. Technically there is no reason why you should need to have a single set. At least not for streams which will only ever be read, or write, but not both. I guess the typical case with most existing APIs is to allow both reading and writing, hence a single set of flags.